### DISCUSSION TITLE
Windows OS version - WDAC Config

### DISCUSSION THREAD

**Discussion by jordfox** [HotCakeX/Harden-Windows-Security#378] [General]
Hi HotCakeX,

I want to test this tool and possibly implement.

However, I am stuck at the first step of import-module... as not running Win 11 24H2.

I am on version 22631.4317 which was released Oct 8 2024, Win 11 23H2.

I have not been offered the Win 11 24H2 update yet, and I am also cautious as I have heard it still has some outstanding bugs.

How to get around this issue and move forward?

Thanks and looking forward to testing.

**Comment by HotCakeX** [comment]
Hi,
I offer certain features of it in an app called AppControl Manager and it should work on 23H2 as the version control is not enforced there: https://github.com/HotCakeX/Harden-Windows-Security/wiki/AppControl-Manager

The app is the future of the WDAC/App Control as it offers a proper GUI to interact with Code Integrity in Windows. It's going to get the rest of the remaining features in the near future.

Btw you don't have to wait for 24H2 to be offered, you can download it yourself and the assistant will guide you: https://www.microsoft.com/en-us/software-download/windows11

You can use it in a Hyper-V VM too if you want.

By the way I've been using and testing Windows 11 24H2 stable build for over a month (and many months before that while it was in release preview state) and it does have one or 2 bugs but it's nothing deal breaker at all. It has many new features, security improvements etc. that you will benefit a lot by using it.

One of the most notable features is the ability to remove policies without the need for restart or deploy signed policies with code integrity turned on without restart.

It has new exploit protections, lots of under the hood security improvements and so on.

**Comment by HotCakeX** [comment]
Hi, I'm releasing an update that will reduce the required version to 23H2 by implementing conditional features based on OS version.
https://github.com/HotCakeX/Harden-Windows-Security/pull/382

Until then you can use the version 0..4.6 by downloading it from PowerShell Gallery

https://www.powershellgallery.com/packages/WDACConfig/0.4.6

And then use the `-SkipVersionCheck` which is supported by every single cmdlet of the WDACConfig module, and it will stay on version 0.4.6.

**Comment by HotCakeX** [comment]
WDACConfig v0.4.8.1 released: https://github.com/HotCakeX/Harden-Windows-Security/releases/tag/WDACConfigv0.4.8

You can continue using it on Windows 11 23H2 again.
